helm-chart-sources/edge/templates/daemonset.yaml

```diff
@@ -121,7 +121,7 @@ spec:
               value: "{{ tpl (print $value) $ }}"
             {{- end }}
             {{- if .Values.extraEnvFrom }}
-            {{- toYaml .Values.envValueFrom | nindent 12 }}
+            {{- toYaml .Values.extraEnvFrom | nindent 12 }}
```

Comment:
@manelpb Can you please add a UT to `edge/tests/daemonset_test.yml`?

Reply:
@bdalpe I don't think unit tests were working for edge bc of the required leader field, i added to all tests and it's working from my machine:

```
helm unittest helm-chart-sources/edge

### Chart [ edge ] helm-chart-sources/edge

 PASS  DaemonSet        helm-chart-sources/edge/tests/daemonset_test.yaml
 PASS  Ingress  helm-chart-sources/edge/tests/ingress_test.yaml
 PASS  RBAC     helm-chart-sources/edge/tests/rbac_test.yaml
 PASS  Secret   helm-chart-sources/edge/tests/secret_test.yaml
 PASS  Service  helm-chart-sources/edge/tests/service_test.yaml

Charts:      1 passed, 1 total
Test Suites: 5 passed, 5 total
Tests:       27 passed, 27 total
Snapshot:    0 passed, 0 total
Time:        107.635792ms
```

Added couple more tests too